themes/mrdrup.sh

```diff
@@ -0,0 +1,33 @@
+export GBT_CARS='Time, Status, Hostname, Dir, Git, Sign'
+
+export GBT_CAR_BG='default'
+
+export GBT_CAR_DIR_FG='111'
+export GBT_CAR_DIR_FM='bold'
+export GBT_CAR_DIR_FORMAT='{{ Dir }}'
+
+export GBT_CAR_GIT_FG='220'
+export GBT_CAR_GIT_FM='bold'
+export GBT_CAR_GIT_FORMAT='{ {{ Icon }} {{ Head }}{{ Stash }}{{ StatusAdded }}{{ StatusDeleted }}{{ StatusModified }}{{ StatusUntracked }}{{ StatusStaged }}{{ Ahead }}{{ Behind }} }'
+
+export GBT_CAR_HOSTNAME_FG='204'
+export GBT_CAR_HOSTNAME_FORMAT='[{{ Host }}]'
+
+export GBT_CAR_SIGN_ADMIN_FG="light_red"
+export GBT_CAR_SIGN_ADMIN_TEXT="#"
+export GBT_CAR_SIGN_FORMAT="{{ Symbol }} "
+export GBT_CAR_SIGN_USER_FG="106"
+export GBT_CAR_SIGN_USER_FM="bold"
+export GBT_CAR_SIGN_USER_TEXT=""
```

Comment:
Use [HEX code](https://github.com/jtyr/gbt/tree/master/sources/gbts#development) instead of the character.